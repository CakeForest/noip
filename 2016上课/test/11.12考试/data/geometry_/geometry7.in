10
34 0 62 32 36 97 10 72 0 56 
3 62 7 10 40 24 56 30 6 68 
10
12 60
62 48
66 90
86 28
56 83
14 55
0 12
0 50
42 0
72 24
