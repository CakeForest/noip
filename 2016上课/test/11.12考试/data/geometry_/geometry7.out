10
10
10
10
10
9
4
7
6
10
